posix-mapper/src/main/webapp/META-INF/context.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='utf-8'?>
+<Context>
+
+    <WatchedResource>WEB-INF/web.xml</WatchedResource>
+
+    <Resource name="jdbc/posix-mapper"
+        auth="Container"
+        type="javax.sql.DataSource"
+        factory="org.apache.tomcat.jdbc.pool.DataSourceFactory" closeMethod="close"
+        minEvictableIdleTimeMillis="60000" timeBetweenEvictionRunsMillis="30000"
+        maxWait="20000"
+        initialSize="0" minIdle="0" maxIdle="${org.opencadc.science-platform.mapping.maxActive}" maxActive="${org.opencadc.science-platform.mapping.maxActive}"
```

Comment:
The keys here and in the README (catalina.properties) should match the java package, so
```
org.opencadc.posix.mapper.???
```
Ah, the README does say that so the fix is just here.